Documentation/ceph-cluster-crd.md

```diff
@@ -1266,6 +1266,27 @@ export ROOK_EXTERNAL_ADMIN_SECRET=AQC6Ylxdja+NDBAAB7qy9MEAr4VLLq4dCIvxtg==
 
 If the Ceph admin key is not provided, the following script needs to be executed on a machine that can connect to the Ceph cluster using the Ceph admin key.
 On that machine, run `cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh`.
+
+> **WARNING**: Before executing the script, RGW_POOL_PREFIX should be exported so as to match the RGW that has been built in Rook. You can get existing RGW by running `ceph osd pool ls`.
```

Comment:
I think some more clarity would be good. Which Rook are you referring to in "the RGW that has been built in Rook"? 

What should users do if there are no pools in the external cluster?

Reply:
I have updated, please review again.